tests/e2e/utils/DriverHelper.ts

```diff
@@ -273,12 +279,23 @@ export class DriverHelper {
                 await element.click();
                 return;
             } catch (err) {
-                if (err instanceof error.StaleElementReferenceError) {
-                    Logger.debug(`DriverHelper.waitAndClik - ${elementLocator} - StaleElementReferenceError - ${err}`);
+                if (err instanceof error.StaleElementReferenceError || (err instanceof error.ElementClickInterceptedError && i !== attempts - 1)) {
```

Comment:
The logic loooks complicated and hard to read.
I would suggest extracting the logic into the method with clear name.

Reply:
Done